.gitmodules

```diff
@@ -10,3 +10,6 @@
 [submodule "contracts-periphery/lib/solmate"]
 	path = contracts-periphery/lib/solmate
 	url = https://github.com/Rari-capital/solmate
+[submodule "contracts-periphery/lib/swap-router-contracts"]
```

Comment:
I don't see this imported anywhere, is this used?

Reply:
Indeed it wasn't, since we are calling the contract address and using our own interface. Removed in https://github.com/ScopeLift/umbra-protocol/pull/341/commits/fc7758fd87a1ada3e4e3d85826f2914a317f7454